src/main/webapp/i18n/de/tutorialGroups.json

```
@@ -0,0 +1,86 @@
+{
+    "artemisApp": {
+        "tutorialGroup": {
+            "deleted": "Übungsgruppe gelöscht"
+        },
+        "tutorialGroups": {
+            "tutorialGroupsButton": "Übungsgruppe"
+        },
+        "manageTutorialGroups": {
+            "title": "Übungsgruppenverwaltung",
+            "tableCaption": "Übungsgruppen für diesen Kurs",
+            "createButton": "Erstelle Übunsgruppe",
+            "columns": {
+                "id": "ID",
+                "title": "Titel",
+                "teachingAssistant": "Tutor",
+                "capacity": "Kapazität",
+                "registeredStudents": "Registrierte Studenten",
+                "isOnline": "Online/Offline",
+                "online": "Online",
+                "offline": "Offline",
+                "location": "Ort",
+                "language": "Sprache",
+                "additionalInfo": "Zusätzliche Informationen"
+            },
+            "deleteButton": {
+                "question": "Bist du sicher, dass du die Übungsgruppe mit dem Titel {{ title }} löschen möchtest?",
+                "typeNameToConfirm": "Titel der Übungsgruppe eingeben, um zu bestätigen"
+            }
+        },
+        "createTutorialGroup": {
+            "title": "Erstelle Übungsgruppe"
+        },
+        "editTutorialGroup": {
+            "title": "Übungsgruppe bearbeiten"
+        },
+        "registeredStudents": {
+            "title": "Registrierte Studenten"
+        },
+        "tutorialGroupForm": {
+            "titleInput": {
+                "label": "Titel",
+                "placeholder": "Gruppe 1: Montag 12:00 - 13:00",
+                "requiredValidationError": "Eine Übungsgruppe muss einen Titel haben!",
+                "maxLengthValidationError": "Der Titel darf maximal 255 Zeichen lang sein!"
+            },
+            "teachingAssistantInput": {
+                "label": "Tutor",
+                "placeholder": "Wähle einen Tutors aus, der für diese Gruppe zuständig ist",
+                "loadingText": "Lade Tutoren...",
+                "notFoundText": "Keine Tutoren gefunden in diesem Kurs",
+                "requiredValidationError": "Ein Tutor muss für diese Gruppe ausgewählt werden!",
+                "taHelp": "Du kannst suchen indem du in dieses Feld tippst"
+            },
+            "additionalInformationInput": {
+                "label": "Zusätzliche Informationen",
+                "placeholder": "Zusätzliche Informationen zu dieser Gruppe",
+                "maxLengthValidationError": "Die zusätzlichen Informationen dürfen maximal 2000 Zeichen lang sein!"
```

```suggestion
                "maxLengthValidationError": "Die zusätzlichen Informationen dürfen maximal {{limit}} Zeichen lang sein!"
```
See the English translation file

---

Done in 82a5edb5cffebb59da7db909205529cf1416d77b